Python Examples/ReadMe.md

```diff
@@ -0,0 +1,28 @@
+SystemLink Enterprise API Examples
+==============================
+
+This directory holds self-contained python scripts for the
+SystemLink Enterprise APIs. 
+
+Running an Example
+------------------
+
+Unless otherwise specified by the example's README, each example is run in the
+same way:
+
+1. Clone _or_ download and extract the [repository source](https://github.com/ni/systemlink-enterprise-examples/archive/master.zip).
+2. Install the [Python SDK](https://www.python.org/downloads/).
```

Comment:
Is this format derived from previous examples? Consider whether should we mention version details

Reply:
This format was taken from the Dotnet examples.

Reply:
Mentioned the version as 3.8 or higher